distributed/distributed.yaml

```diff
@@ -274,6 +274,7 @@ distributed:
       max-history: 100
       nframes: 0
       ignore-modules:
+        - _pytest
```

Comment:
Just for my own understanding, why `_pytest` and not `pytest`?

Reply:
https://github.com/pytest-dev/pytest/blob/main/src/pytest/__init__.py doesn't have any function defs and so won't appear in any frames

Reply:
all the function defs are in https://github.com/pytest-dev/pytest/tree/main/src/_pytest

Reply:
TIL 🙂